json_rpc/clients/httpclient.nim

```diff
@@ -84,8 +102,7 @@ proc connect*(client: RpcHttpClient, url: string)
     raise newException(RpcAddressUnresolvableError, client.httpAddress.error)
 
 proc connect*(client: RpcHttpClient, address: string, port: Port) {.async.} =
-  let addresses = resolveTAddress(address, port)
-  if addresses.len == 0:
-    raise newException(RpcAddressUnresolvableError, "Failed to resolve address: " & address)
-  ok client.httpAddress, getAddress(addresses[0])
-
+  let url = "http://" & address & ":" & $port
```

Comment:
Constructing an `Uri` value will be more efficient than constructing and parsing a string.

Reply:
865bb338438dfe0f62d21e3df3e9efb46da9a20c